7/Constructor1/script.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+let mainTextAddY = document.getElementById("mainTextAddY");
+let mainTextAddN = document.getElementById("mainTextAddN");
+let mainText = document.getElementById("mainText");
+
+mainTextAddY.onclick = function(){
+    mainText.removeAttribute('disabled');
+};
```

Comment:
Please try to use ```addEventListener```:
```javascript
const onMainTextYClick = () => mainText.removeAttribute('disabled');
const Events = { CLICK: 'click' , ... /* other event names */}
mainTextAddY.addEvetListener(Events.CLICK, onMainTextYClick)
```
https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/addListener

Reply:
Done